### DISCUSSION TITLE
30N30P_MultiElement_Airfoil

### DISCUSSION THREAD

**Discussion by hosshvsi** [mdolab/dafoam#645] [Bug]
Hello everyone,
I just wanted to run the multi element airfoil tutorial, I installed v3.1.1 and then mount it 
after couple of seconds, I faced with error message. It was wired because I didn't change anything (mesh setup or running script).
would you please help me in case to address the issue?

[logOpt.txt](https://github.com/user-attachments/files/15751185/logOpt.txt)

**Comment by friedenhe** [comment]
I just ran the case without a problem. Did you run `Allclean.sh` before running the case?

**Reply by hosshvsi** [reply]
yes, I did 
after reaching 2,000 iterations, apparently, I will be facing with an error regarding Primal Residual:

Primal min residual 2.749081039355476e-06
did not satisfy the prescribed tolerance 1e-08
Primal solution failed!

 to address that issue, I increased the number of iterations in controlDict to 10,000 and also decrease Primal Residual from 1e-8 into 1e-6, then it was working but after 2 hours this error which I attached shows up.
 
 Im suing v.3.1.2 first ./preProcessing.sh and then mpirun -np 4 python runScript.py
 
[DAFoam_error.txt](https://github.com/user-attachments/files/15777861/DAFoam_error.txt)

**Comment by friedenhe** [comment]
The case finished successfully. The "error" you mentioned is a warning.

**Reply by hosshvsi** [reply]
oh, great news, thank you so much for your support ')
